performance/client/src/__tests__/primary/ExplorerAPIs/get-unixfiles-list.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright IBM Corporation 2020
+ */
+
+import WrkTestCase from "../../../testcase/wrk";
+import { getBasicAuthorizationHeader } from "../../../utils";
+
+class ExplorerApiUnixFilesListTest extends WrkTestCase {
+  fetchZoweVersions = true;
+
+  name = "Test explorer data sets api endpoint /unixfiles";
+  endpoint = '/api/v2/unixfiles';
```

Comment:
Without filter, what this endpoint will return? Is that the file list of user home directory? One thing please keep in mind, sometimes the user may not have home directory. We can pick a folder always exists like /etc.

Reply:
I modified this test to specifically pull the contents of the `/SYSTEM/etc` directory, as that should exist on all systems.